aries-site/src/layouts/main/Layout.js

```diff
@@ -21,12 +23,39 @@ export const Layout = ({ children, title, isLanding }) => {
             margin="auto"
           >
             <Head title={title} />
-            <Header showLinks={!isLanding} />
-            {!isLanding ? (
+            <Header
+              showLinks={!isLanding && !isNavPage}
```

Comment:
Expanding the number and complexity of conditionals starts to make it difficult to know what the `<Layout />` component is going to return. I think we are fine to move forward as is for now to keep things moving, but would like to revisit how we are approaching page layouts in the future --> I'd like to discuss moving in a direction where the consumer of a page layout is explicitly stating which type of page layout they would like (e.g. Content Page Layout, Landing Page Layout, Navigation Page Layout, etc.).

With the current conditionals, I think this is manageable, but introducing any more becomes more difficult to read and reason about, increases the number of edge cases, and increases risk of introducing bugs.

Reply:
I agree, this is a concept we should revisit.